src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.ps1

```diff
@@ -494,4 +494,60 @@ function Test-ShortTermRetentionPolicy
 	{
 		Remove-ResourceGroupForTest $rg
 	}
+}
+
+function Test-CopyLongTermRetentionBackup
+{
+	# MANUAL INSTRUCTIONS
+	# Create a server and database and fill in the appropriate information below
+	# Set the weekly retention on the database so that the first backup gets picked up, for example:
+	# Set-AzSqlDatabaseLongTermRetentionPolicy -ResourceGroup $resourceGroup -ServerName $serverName -DatabaseName $databaseName -WeeklyRetention P1W
+	# Wait about 18 hours until it gets properly copied and you see the backup when run get backups, for example:
+	# Get-AzSqlDatabaseLongTermRetentionBackup -Location $locationName -ServerName $serverName -DatabaseName $databaseName -ResourceGroupName $resourceGroup
+	$resourceGroupName = "testrg"
```

Comment:
change this to two variables $sourceResourceGroupName, $targetResourceGroupName, the value can be the same